samples/azure-monitor/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Scrape Prometheus Metrics Endpoints
 
-SAS Viya components are natively instrumented to expose a Prometheus-compatible
+SAS Viya platform components are natively instrumented to expose a Prometheus-compatible
```

Comment:
There is a requirement for all components within SAS Viya (all flavors) to be instrumented to permit monitoring in this way.  Therefore, I believe the addition of "platform" here is inappropriate.